dgraph/graphql/src/api.js

```diff
@@ -0,0 +1,34 @@
+import { ApolloServer } from 'apollo-server-fastify';
+import fastify from 'fastify';
+import { typeDefs } from './schema';
+import { dgraph } from './data';
+
+const resolvers = {
+  Query: {
+    searchByName: async (_source, _args, { dataSources }) => {
+      const dgraphQuery = `{
+        query(func: has(code)) @filter(regexp(description, /^${_args.text}.*$/i) AND eq(type, "unit_of_use")) {
```

Comment:
I have no idea why.. but for me, I had to rearrange the order before I got results:

```suggestion
        query(func: regexp(description, /^${_args.text}/i)) @filter(has(code) AND eq(type, "unit_of_use")) {
```

Reply:
So strange! This rearranged one still works for me so I'm happy to go with this one :) I did get a warning about needing the trigram index on description for it to work so will update the schema to reflect this.